src/cli.ts

```diff
@@ -16,6 +16,41 @@ interface Repository {
     directory?: string
 }
 
+/**
+ * Flag indicating whether to include test tooling in the project, namely a test
+ * framework (mocha), extension API stubs, and a sample test.
+ *
+ * TODO: in the future, make this an option chosen by the user.
+ */
+const testsEnabled = true
+
+/**
+ * Module dependencies to add to package.json as devDependencies
+ */
+const devDependencies: string[] = [
+    'sourcegraph',
+    'typescript',
+    'parcel-bundler',
+    'eslint',
+    '@sourcegraph/eslint-config',
+    '@sourcegraph/tsconfig',
+    'lnfs-cli',
+    'mkdirp',
+]
+
+/**
+ * Module dependencies that are used for automated tests. Used if `testsEnabled` is true.
+ */
+const devDependenciesForTests: string[] = [
+    '@sourcegraph/extension-api-stubs',
+    'mocha',
```

Comment:
We may wanna add `nyc` for coverage

Reply:
Good idea and I turned it into an issue #154 to come back to it.